### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by ericnyamu** [debops/ansible-ferm#43]
Hi.here's my situation.i have some computers that are behind a nat.i would like to pass this 'natting ip' through a vpn,so that all the pc behind the nat can be using the vpn.how should the iptables commands look like.The vpn interface is tun0 and the natting interface is eth0.thanks

**Comment by drybjed** [comment]
Hmm, let me rephrase that to see if I understood correctly:

- you have a NAT network on `192.168.0.0/24` (or what have you) behind `eth0` interface;

- you have a VPN connection on `10.10.10.1/24` at `tun0` interface;

- you want hosts from NAT network to be able to connect to VPN network?

Or is it that `10.10.10.1` IP address should be visible on the NAT network as say, `192.168.0.32`? If this is correct, you can do it like so:
```
ferm_host_rules:
  - type: 'dmz'
    name: 'forward_vpn_to_nat'
    public_ip: '192.168.0.32'
    private_ip: '10.10.10.1'
```
I haven't tested to be sure, but that should forward all traffic on any port/protocol from `192.168.0.32` to `10.10.10.1`. Of course interfaces need to be configured with these IP addresses, you can do that using `debops.ifupdown`.
